wbb/modules/rice_DISABLED.py

```diff
@@ -1,11 +1,13 @@
 from wbb import app
 from wbb.utils import cust_filter
 from pyrogram import filters
+
+
 __MODULE__ = "Rice"
-__HELP__ = "/rice - To Forward Your Linux Rice To DE_WM's Rice Gallery"
+__HELP__ = "/rice - To Forward Your Linux Rice To DE_WM's Rice Gallery only works in [DE_WM](t.me/de_wm)"
 
 
-@app.on_message(filters.chat('de_wm') & cust_filter.command(commands=("rice")))
+@app.on_message(filters.chat('DE_WM') & cust_filter.command(commands=("rice")))
```

Comment:
> await message.reply_to_message.forward(*****)

change this chat_id to the channel's username

Reply:
this is username

Reply:
I mean "forward", where the message was forwarded, that channel.

Reply:
fixed